tests/utils.go

```diff
@@ -88,21 +98,14 @@ const (
 )
 
 const (
-	labelISCSIPod         = "iscsi-demo-target"
-	labelISCSIWithAuthPod = "iscsi-auth-demo-target"
-	labelNFSPod           = "nfs-server-demo"
+	labelNFSPod = "nfs-server-demo"
```

Comment:
This NFS pod doesn't exist anymore. Whatever references it should be able to be removed safely at this point. It's just dead code.